navit/maptool/itembin_slicer.c

```diff
@@ -0,0 +1,853 @@
+/**
```

Comment:
This brakes doxygen, plaease use `/*` instead for the top head license header, because otherwise its treaded as the description of the file

Reply:
Sure there is no other reason for breaking?
I copied the license header from itembin.c. And it is similar on most other files in "maptool" at least.
I can remove that additional asterix, no problem. But do we need to remove it from the other files as well?

Btw: Where can I check the doxygen result? In CI artefacts?

Reply:
> Where can I check the doxygen result? In CI artefacts?

http://doxygen.navit-project.org/



> But do we need to remove it from the other files as well?

yes